{
  "configurations": {
    "CVE_data_version": "4.0",
    "nodes": []
  },
  "cve": {
    "CVE_data_meta": {
      "ASSIGNER": "cve@mitre.org",
      "ID": "CVE-2020-15142"
    },
    "data_format": "MITRE",
    "data_type": "CVE",
    "data_version": "4.0",
    "description": {
      "description_data": [
        {
          "lang": "en",
          "value": "In openapi-python-client before version 0.5.3, clients generated with a maliciously crafted OpenAPI Document can generate arbitrary Python code. Subsequent execution of this malicious client is arbitrary code execution."
        }
      ]
    },
    "problemtype": {
      "problemtype_data": [
        {
          "description": []
        }
      ]
    },
    "references": {
      "reference_data": [
        {
          "name": "https://github.com/triaxtec/openapi-python-client/blob/main/CHANGELOG.md#053---2020-08-13",
          "refsource": "MISC",
          "tags": [],
          "url": "https://github.com/triaxtec/openapi-python-client/blob/main/CHANGELOG.md#053---2020-08-13"
        },
        {
          "name": "https://github.com/triaxtec/openapi-python-client/commit/f7a56aae32cba823a77a84a1f10400799b19c19a",
          "refsource": "MISC",
          "tags": [],
          "url": "https://github.com/triaxtec/openapi-python-client/commit/f7a56aae32cba823a77a84a1f10400799b19c19a"
        },
        {
          "name": "https://github.com/triaxtec/openapi-python-client/security/advisories/GHSA-9x4c-63pf-525f",
          "refsource": "CONFIRM",
          "tags": [],
          "url": "https://github.com/triaxtec/openapi-python-client/security/advisories/GHSA-9x4c-63pf-525f"
        },
        {
          "name": "https://pypi.org/project/openapi-python-client/",
          "refsource": "MISC",
          "tags": [],
          "url": "https://pypi.org/project/openapi-python-client/"
        }
      ]
    }
  },
  "impact": {},
  "lastModifiedDate": "2020-08-14T18:08Z",
  "publishedDate": "2020-08-14T17:15Z"
}